core/node/da_clients/src/eigen/eigenda-integration.md

```diff
@@ -113,6 +113,34 @@ You may enable observability here if you want to.
 zkstack server --chain eigen_da
 ```
 
+### Data Availability Grafana Metrics
+
+1. Get the running port of the eigen_da chain in the `chains/eigen_da/configs/general.yaml` file:
+
+```yaml
+prometheus:
+  listener_port: 3414 # <- this is the port
+```
+
+(around line 108)
+
+Then modify the `era-observability/etc/prometheus/prometheus.yml` with the retrieved port:
+
+```yaml
+scrape_interval: 5s
+honor_labels: true
+static_configs:
+  - targets: ['host.docker.internal:3312'] # <- change this to the port
+```
```

Comment:
We should indicate which one exactly to change this, it should be the one named zksync